source/administration/devops-command-center.rst

```diff
@@ -24,29 +24,29 @@ Starting incidents
 The incident channel is the central place for discussion related to the incident. You can start an incident in one of three ways:
 
 - Use the slash command */incident start* from any channel.
-- Select **! Incidents** from the plugin menu in the channel header.
+- Select **!** from the channel header.
 - Use the context menu of a post and select **Start incident**.
 
-When you create an incident, the name provided is applied to the new incident channel that is created. You can also select an optional playbook. 
+When you create an incident, the name provided is applied to the new incident channel that is created. You can also select an optional playbook.
 
-- If a playbook is selected, a corresponding checklist is used as a template to start the incident. 
+- If a playbook is selected, a corresponding checklist is used as a template to start the incident.
 - If no playbook is selected, the incident starts with an empty checklist.
 
 The creator of an incident automatically becomes the incident commander, responsible for managing the incident.
 
 The commander of the incident can:
 
-- Select a playbook when starting an incident
-- Create checklists
-- Invite people to the channel
-- Hand over the commander role to another channel member
+- Select a playbook when starting an incident.
+- Create checklists.
+- Invite people to the channel.
+- Hand over the commander role to another channel member.
```

Comment:
I *never* know what the punctuation in lists should be. We can discuss in the Language channel in Mattermost! :)

Reply:
Great catch. In cases like this where there are shorter and longer sentences, I use periods for each line for consistency. For lists with single words/short phrases I don't use punctuation. I have noted that this is missing from our style guide so I'll add that in, thanks. 

As an aside, [this article](https://intelligentediting.com/blog/how-to-punctuate-bulleted-and-numbered-lists/#:~:text=There%20is%20no%20single%20right,take%20different%20forms%20of%20punctuation.&text=a%20semicolon%20otherwise) has some guidance.

Reply:
Oh gosh, thank you for that link!